### DISCUSSION TITLE
could soneo post the results here, I cannot visit linkedin

### DISCUSSION THREAD

**Discussion by l1t1** [duckdb/duckdb#8555] [General]
https://by.linkedin.com/posts/mimounedjouallah_tpch-sf10-using-duckdb-vs-snowflake-bigquery-activity-7040907928958095360-08p2

**Comment by darthf1** [comment]
It leads to this website:

https://www.architecture-performance.fr/ap_blog/tpch-sf10-using-duckdb-vs-snowflake-bigquery-singlestore-and-databricks/

**Comment by l1t1** [comment]
thanks a lot
